joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/Main.scala

```diff
@@ -28,7 +29,11 @@ private object Frontend {
       programName("php2cpg"),
       opt[String]("php-ini")
         .action((x, c) => c.copy(phpIni = Some(x)))
-        .text("php.ini path used by php-parser. Defaults to php.ini shipped with Joern.")
+        .text("php.ini path used by php-parser. Defaults to php.ini shipped with Joern."),
+      opt[Seq[String]]("exclude-overrides")
```

Comment:
can we split this in two? I think from an end-user's perspective they either want to disable the default exclusions or add additional exclusions, but rarely both at the same time.